src/components/MediaHorizontalList.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <div>
+        <b-container class="container">
+            <div v-for="mediaObj in mediaObjs" :key='mediaObj.ids.imdb'>
```

Comment:
Is this `<div>` necessary? Put v-for on MediaTile instead

Reply:
No, turns out the `<div>` isn't necessary. I'll remove that. 

I'm not sure I understand the `v-for` suggestion. Could you elaborate? I don't think I could remove that `<div>` on line 4 if that's what you're asking.

Reply:
To clarify, the `v-for` attribute can be on *any* element, not just divs. Hence you could place it (along with the `:key` attribute) on the MediaTile element instead and have exactly the same effect. The `<div>` would then be unnecessary.